db.go

```diff
@@ -477,17 +493,13 @@ func (db *DB) reload() (err error) {
 			return errors.Wrapf(err, "unexpected corrupted block %s", c)
 		}
 	}
-	// Load new blocks into memory.
+	// allBlocks := make([]*Block, 0)
+	// Get storage blocks total size
 	for _, dir := range dirs {
 		meta, err := readMetaFile(dir)
 		if err != nil {
 			return errors.Wrapf(err, "read meta information %s", dir)
 		}
-		// Don't load blocks that are scheduled for deletion.
```

Comment:
I think the logic after this is wrong. The `deleteable` blocks are left over after the compaction has merged these so these should be excluded from the check.
Anyway I want to try to simplify this as it is a bit difficult to follow.

Reply:
I don't really remember why or how that got deleted, definitely should still be in there.